tests/phpunit/Mutation/FileMutationGeneratorTest.php

```diff
@@ -128,7 +125,7 @@ public function test_it_attempts_to_generate_mutations_for_the_file_if_covered_o
         LineCodeCoverage $codeCoverage,
         string $expectedFilePath
     ): void {
-        $extraVisitors = [2 => new FakeVisitor()];
+        $extraVisitors = [new FakeIgnores()];
```

Comment:
Reviewing this class, the `$mutationGenerator` property of the test case should be used in `test_it_attempts_to_generate_mutations_for_the_file_if_covered_or_not_only_covered_code` and `test_it_skips_the_mutation_generation_if_checks_only_covered_code_and_the_file_has_no_tests` and the created mocks can be the ones of the test case as well